pkg/cover/report_test.go

```diff
@@ -180,7 +180,17 @@ func generateReport(t *testing.T, target *targets.Target, test Test) ([]byte, []
 	}
 	defer os.RemoveAll(dir)
 	bin := buildTestBinary(t, target, test, dir)
-	rg, err := MakeReportGenerator(target, "", dir, dir, dir)
+	subsystem := []Subsystem{
+		{
+			Name: "sound",
+			Paths: []string{
+				"sound",
+				"techpack/audio",
+			},
+		},
+	}
+
+	rg, err := MakeReportGenerator(target, "", dir, dir, dir, subsystem)
```

Comment:
Also please call the new functions below where the test calls DoHTML/DoCSV. That will at least ensure that the code does not crash on some stupid nil deref after some refactoring.

Reply:
partiall fixed, added these two calls, but I didn't checked the DoCSVFiles result as I don't have the main string.